descartes_light/edge_evaluators/include/descartes_light/edge_evaluators/normalized_edge_evaluator.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <descartes_light/core/edge_evaluator.h>
+
+namespace descartes_light
+{
+/**
+ * @brief Edge evaluator which normalizes the cost returned by an internal edge evaluator on [0.0, 1.0]
+ */
+template <typename FloatType>
+class NormalizedEdgeEvaluator : public EdgeEvaluator<FloatType>
+{
+public:
+  NormalizedEdgeEvaluator(typename EdgeEvaluator<FloatType>::ConstPtr evaluator,
+                          const FloatType min,
+                          const FloatType max);
+
+  virtual std::pair<bool, FloatType> evaluate(const State<FloatType>& start,
+                                              const State<FloatType>& end) const override;
+
+private:
+  typename EdgeEvaluator<FloatType>::ConstPtr evaluator_;
+  const FloatType min_;
+  const FloatType max_;
```

Comment:
Recommend not making this const. I believe the compiler will delete the assignment operator for a class if it contains const member variables. The `evaluate` function is const so it should not be possible to change the member variables during execution.

Reply:
Is copy-assignment critical for this class? I like making members that shouldn't change `const` so I don't have to worry much about unintentionally changing them in a function

Reply:
I just tested this in Godbolt without any issues. I think it is fine as is. If we run into issues we can address them then.